cmake/kokkos_enable_options.cmake

```diff
@@ -58,6 +59,7 @@ ELSE()
 ENDIF()
 KOKKOS_ENABLE_OPTION(CUDA_LAMBDA ${LAMBDA_DEFAULT} "Whether to activate experimental lambda features")
 
+KOKKOS_OPTION(SKIP_VALIDATION OFF BOOL "Whether to skip certain validation checks while testing")
```

Comment:
Hell no.  They are enough options already and this certainly should not be exposed to the user.  I'd rather only emit warnings all the time and not stop the generation than have the option.

Reply:
So what is the plan for running these validation scripts?  I can remove this as a user-visible option (i.e. it won't show up in the cache unless you give it).  Otherwise the scripts need to be changed.

Reply:
OK I propose the following:  Remove this line and unconditionally emit a warning and turn the offending option off.